Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -2610,14 +2610,14 @@ void TestController::didReceiveSynchronousMessageFromInjectedBundle(WKStringRef
         if (WKStringIsEqualToUTF8CString(subMessageName, "SetPageZoom")) {
             auto* page = mainWebView()->page();
             WKPageSetTextZoomFactor(page, 1);
-            WKPageSetPageZoomFactor(page, doubleValue(dictionary, "ZoomFactor"));
+            WKPageSetPageZoomFactor(page, WKPageGetPageZoomFactor(page) * (booleanValue(dictionary, "ZoomIn") ? 1.2 : (1.0 / 1.2)));
```

Comment:
ZoomMultiplierRatio is now used 4 times.  I think it would be worth keeping its name rather than having a magic number 4 times.